e2e/scripts/generate-project.js

```diff
@@ -62,7 +62,8 @@ program
             'retail-app-demo',
             'retail-app-ext',
             'retail-app-no-ext',
-            'retail-app-private-client'
+            'retail-app-private-client',
+            'retail-react-app-bug-bounty'
```

Comment:
I'm leveraging the generate project script in the e2e directory but it may be a good idea in the future to move this if we continue to use the script for jobs other than the e2e test CI.